src/utils/PropTypes.js

```diff
@@ -5,8 +5,8 @@ export const timeoutsShape =
     ? PropTypes.oneOfType([
         PropTypes.number,
         PropTypes.shape({
-          enter: PropTypes.number,
-          exit: PropTypes.number,
+          enter: PropTypes.number.isRequired,
+          exit: PropTypes.number.isRequired,
           appear: PropTypes.number,
         }).isRequired
       ])
```

Comment:
Move `isRequired` from `shape` to `oneOfType`. Totally optional, but would be awesome 😎

Reply:
Ok, you're right, i remove isRequired.